api/main.py

```diff
@@ -1,18 +1,12 @@
 from fastapi import FastAPI
 from fastapi.responses import RedirectResponse
 from src.routers.health import health_router
-from src.routers.mtcars import data_output
 from src.routers.endpoints import endpoints
 from fastapi.middleware.cors import CORSMiddleware
-from os import getenv
-from dotenv import load_dotenv
+
 import uvicorn
 import tomllib
 
-# import .env settings
-load_dotenv()
-API_PORT = int(getenv("API_PORT", 8080))
-
```

Comment:
This code shouldn't be removed. It was added in #65 to handle port mapping using an external ENVVAR. 

This is likely where the Docker GH action failure is coming from :)

Reply:
fixed